packages/modules/cra-template-web-viewer/template/src/history.ts

```diff
@@ -2,7 +2,11 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
 
-import { createBrowserHistory } from "history";
+import createBrowserHistory from "history/createBrowserHistory";
```

Comment:
any real benefit of changing this import?

Reply:
with the change to the tsconfig
`"baseUrl": "src"`

It introduces a circular dependency. The baseUrl addition is useful for when we add our common/local modules to the template, its all based off the src folder rather than its location in the module.

Reply:
This causes this error/warning in the console:

![image](https://user-images.githubusercontent.com/6283674/135451227-c28c6277-4ae9-45af-9581-4f278fa99d84.png)

Guess I'm not following how this import causes a circular dependency?